Utilities/CreateServicePrincipal.ps1

```diff
@@ -30,17 +30,17 @@ function New-SWRandomPassword {
         New-SWRandomPassword -InputStrings abc, ABC, 123 -PasswordLength 4
         3ABa
 
-        Generates a password with a length of 4 containing atleast one char from each InputString
+        Generates a password with a length of 4 containing at least one char from each InputString
     .EXAMPLE
         New-SWRandomPassword -InputStrings abc, ABC, 123 -PasswordLength 4 -FirstChar abcdefghijkmnpqrstuvwxyzABCEFGHJKLMNPQRSTUVWXYZ
         3ABa
 
-        Generates a password with a length of 4 containing atleast one char from each InputString that will start with a letter from
+        Generates a password with a length of 4 containing at least one char from each InputString that will start with a letter from
         the string specified with the parameter FirstChar
     .OUTPUTS
         [String]
     .NOTES
-        Written by Simon Wåhlin, blog.simonw.se
+        Written by Simon Wahlin, blog.simonw.se
```

Comment:
Wait, if this comes from someone else's script, I am not sure if we put Apache license. Need to check with CELA.